README.md

```diff
@@ -1,241 +1,296 @@
+   containerized environment and/or for expert users.
 
-  If not, add it with `sudo adduser $USER render`, logout and log back in, and check
-  again.
+   To not alter the `apt`-based version tree too much and risk other compatibility
+   issues, the recommended workaround consists in identifying the version that is
+   officially supported by your OS (use [packages.ubuntu.com](https://packages.ubuntu.com/search?keywords=intel-opencl-icd&searchon=names))
+   then find the corresponding build from the [Intel release page on github](https://github.com/intel/compute-runtime/releases)
+   and follow the instruction from the release page, e.g for `jammy`:
 
-- Install recent versions of the [runtime libraries](https://github.com/intel/compute-runtime/releases) (not yet available
-  as conda packages)
+   ```bash
+    mkdir neo
+    cd neo
+    wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-gmmlib_22.0.2_amd64.deb
+    wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.10840/intel-igc-core_1.0.10840_amd64.deb
+    wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.10840/intel-igc-opencl_1.0.10840_amd64.deb
+    wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-opencl-icd-dbgsym_22.14.22890_amd64.ddeb
+    wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-opencl-icd_22.14.22890_amd64.deb
+    cd ../ && rm -Rf neo
+    ```
 
-For more in-depth information, you can refer to the [guides for system configuration for Intel hardware and software](https://www.intel.com/content/www/us/en/developer/articles/system-requirements/intel-oneapi-dpcpp-system-requirements.html) and to the discussions in this github issue https://github.com/IntelPython/dpnp/issues/1149
+- **oneAPI level zero loader**: alternatively, or in addition, the oneAPI level zero
+  backend can be used. This backend is more experimental, and is sometimes preferred
+  over opencl. Source and `deb` archives are available
+  [here](https://github.com/oneapi-src/level-zero/releases).
 
+###### Give permissions to submit GPU workloads
 
-The `dpctl.lsplatform()` can also list version informations on your SYCL
-runtime environment:
+Non-root users might lack permission to access the GPU device to submit workloads. Add
+the current user to the `video` group and/or `render` group:
 
+```bash
+sudo usermod -a -G video $USER
+sudo usermod -a -G render $USER
 ```
-python -c "import dpctl; dpctl.lsplatform()"
-```
-
-##### Install numba-dpex from source (2/2)
 
-Install and activate the Intel oneAPI DPC++ compiler shipped with the [Intel oneAPI base toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html).
-
-For instance, in Ubuntu, once the apt repo has been configured:
+##### Setup a conda environment for numba-dpex (2/2)
 
+To setup a [conda](https://docs.conda.io/en/latest/) conda environment, run first
```

Comment:
I would be in favor of making instructions explicit to users so that they can be autonomous and understand potential errors better.
```suggestion
You can setup a [`conda`](https://docs.conda.io/en/latest/) environment, and
install dependencies (`numba-dpex` and `intel::dpcpp_linux-64`) distributed
on the `conda-forge`, `intel` and experimental `dppy/label/dev` channels with:
```